de.dlr.sc.virsat.model.calculation.test/src/de/dlr/sc/virsat/model/calculation/compute/AEquationTest.java

```diff
@@ -57,34 +43,16 @@
 	protected EquationSectionVirSatAwareXtextResourceSet esResourceSet;
 	protected EquationSectionXtextResource esResource;
 	protected List<EObject> contents;
-	protected EditingDomain editingDomain;
-	
 	
 	@Before 
 	public void setUp() throws Exception {
 		contents = new ArrayList<>();
-		
 		UserRegistry.getInstance().setSuperUser(true);
-		
-		ComposedAdapterFactory adapterFactory = new ComposedAdapterFactory(ComposedAdapterFactory.Descriptor.Registry.INSTANCE);
-		adapterFactory.addAdapterFactory(new DVLMResourceItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new DVLMDVLMItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new DVLMStructuralItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new GeneralItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new ConceptsItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new DVLMCategoriesItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new PropertydefinitionsItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new PropertyinstancesItemProviderAdapterFactory());
-		adapterFactory.addAdapterFactory(new ReflectiveItemProviderAdapterFactory());
-		BasicCommandStack commandStack = new BasicCommandStack() {
-			@Override
-			public void execute(Command command) {
-				command.execute();
-			}
-		};
-		
-		editingDomain = new AdapterFactoryEditingDomain(adapterFactory, commandStack, new HashMap<Resource, Boolean>());
+	}
 	
+	@After

```

Comment:
The super user should always be deactivated after each test. Why is it removed here?

Reply:
Also agree here, which is why I added the tearDown method to set the super user to false.

Reply:
Oh yes i see. it god added :see_no_evil: .